Kernel/FileSystem/Ext2Base.h

```diff
@@ -0,0 +1,307 @@
+/*
+ * Copyright (c) 2022, iProgramInCpp <iprogramincpp@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+namespace Kernel::Ext2 {
+// The second extended filesystem constants/structures
+
+constexpr u32 ext4_feature_incompat_64bit = (1 << 7);
+constexpr u32 max_name_len = 255;
+constexpr u16 super_magic = 0xEF53;
+
+// Constants relative to the data blocks
+constexpr u32 num_direct_blocks = 12;
+
+// FIXME: Give these variables a better name.
+constexpr u32 indirect_block = num_direct_blocks,
+              dindirect_block = indirect_block + 1,
+              tindirect_block = dindirect_block + 1,
+              num_blocks = tindirect_block + 1;
```

Comment:
Maybe we want to prefix every line with `constexpr u32` here. At least I don't recall seeing this style of declaration anywhere else in Serenity.

Reply:
Sure. I put those in such a chain because each value is related to the previous and I didn't want to repeat `constexpr u32`.